src/modals/SettingsModal/CustomThemeForm.js

```diff
@@ -341,13 +352,40 @@ export default function CustomThemeForm() {
               label="hide"
               name="thumbCounterDisplay"
               id="thumbCounterDisplay-hide"
-              checked={customTheme.thumbCounterDisplay === "none"}
+              checked={currentCustomTheme.thumbCounterDisplay === "none"}
               onChange={(e) =>
                 setCustomThemeProp("thumbCounterDisplay", "none")
               }
             />
           </Col>
         </Form.Group>
+        {/*<Form.Group as={Row}>*/}
+        {/*  <Form.Label as="legend" column sm={3} className="pt-0">*/}
+        {/*    EntiTree Search bar*/}
+        {/*  </Form.Label>*/}
+        {/*  <Col sm={9}>*/}
+        {/*    <Form.Check*/}
+        {/*      type="radio"*/}
+        {/*      label="show"*/}
+        {/*      name="isInIframe"*/}
+        {/*      id="isInIframe-show"*/}
+        {/*      checked={currentCustomTheme.isInIframe === false}*/}
+        {/*      onChange={(e) =>*/}
+        {/*        setCustomThemeProp("isInIframe", false)*/}
+        {/*      }*/}
+        {/*    />*/}
+        {/*    <Form.Check*/}
+        {/*      type="radio"*/}
+        {/*      label="hide"*/}
+        {/*      name="isInIframe"*/}
+        {/*      id="isInIframe-hide"*/}
+        {/*      checked={currentCustomTheme.isInIframe === true}*/}
+        {/*      onChange={(e) =>*/}
+        {/*        setCustomThemeProp("isInIframe", true)*/}
+        {/*      }*/}
+        {/*    />*/}
+        {/*  </Col>*/}
+        {/*</Form.Group>*/}
```

Comment:
?

Reply:
I was thinking to hide the headers, but once there hidden, how to show them again😂😂😂